tests/cases/DI/MenuExtensionTest.php

```diff
@@ -30,6 +30,15 @@ public function testDI(): void
 		Assert::type(MenuContainer::class, $dic->getService('menu.container'));
 		Assert::type(MenuComponentFactory::class, $dic->getService('menu.component.factory'));
 		Assert::type(MenuComponent::class, $dic->getService('menu.component.factory')->create('default'));
+
+		$container = $dic->getService('menu.container');
+		/** @var \Contributte\MenuControl\IMenu $menu */
+		$menu = $container->getMenu('default');
+
+		$item = $menu->getItem('Homepage');
+
+		Assert::type('string', $item->getDataItem('icon'));
+		Assert::type('array', $item->getDataItem('structured'));
```

Comment:
Should be in separate test function, not in `testDI`

Reply:
Well, it is part of DI because otherwise it would fail to compile 🤷

Reply:
Yes, but this is not DI test. You can call `$dic = $this->createContainer();` in another function. Called `testDataItems()` or something :-)